README.md

```diff
@@ -1,2 +1,33 @@
-# OsedeaTechLinks
-A repository of shared knowledge in the form of links to various topics and teachings in Technology and Science.
+# Osedea Tech Links
+A repository of shared knowledge in the form of links to various topics and teachings in Technology, News and Science.
+
+Structure:
+* Primary Folder - "Links"
+    * Sub Directories for each topic/language/framework
+    * Example:
```

Comment:
I feel like it's unclear that this is an example/a guide for how to navigate this repo